### PR TITLE
Updates for Tutorial5

### PR DESCRIPTION
**Pull request by tir38**

### CODE REVIEW

File: samples/tutorial/Tutorial5.md [hunk 1]
```diff
@@ -53,6 +53,35 @@ class WelcomeWorkflowTest {
 }
 ```
 
+Update `build.gradle` to include two test dependencies
```
Comment (tir38) [inline samples/tutorial/Tutorial5.md:56]: two missing steps

File: samples/tutorial/Tutorial5.md [hunk 2]
```diff
@@ -353,11 +382,11 @@ class RootWorkflowTest {
         // Now, validate that there is a single item in the BackStackScreen, which is our welcome
         // screen.
         .render { rendering ->
-          val backstack = (rendering as BackStackScreen<*>).frames
+          val backstack = rendering.frames
```
Comment (tir38) [inline samples/tutorial/Tutorial5.md:385]: unnecessary cast

**Review by steve-the-edwards — APPROVED** [review]
Thanks!

**Comment by tir38** [comment]
@steve-the-edwards i think you have to merge this. I don't have write acess

**Pull request merged**